am-lib/src/main/java/uk/gov/hmcts/reform/amlib/internal/utils/SecurityClassifications.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.hmcts.reform.amlib.internal.utils;
+
+import uk.gov.hmcts.reform.amlib.enums.SecurityClassification;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+import static java.util.stream.Collectors.toSet;
+
+public final class SecurityClassifications {
+
+    private SecurityClassifications() {
+        throw new UnsupportedOperationException("Constructing utility class is not supported");
+    }
+
+    /**
+     * Builds a set of security classifications including and below a given hierarchy.
+     *
+     * @param securityClassificationForRole integer value of security classification defined in
+     * {@link SecurityClassification}
+     * @return a set of security classifications
+     */
+    public static Set<SecurityClassification> getVisibleSecurityClassifications(int securityClassificationForRole) {
```

Comment:
Argument name change is still outstanding. As I pointed earlier mentioning role does not make much sense in generic utility class.

Reply:
changed to hierarchy